profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.module

```diff
@@ -0,0 +1,747 @@
+ */
+function nexteuropa_trackedchanges_get_human_name($data, array $options, $name, $type, $info) {
+  $human_name = $name;
+  switch ($name) {
+    case 'rel_entity_state_human_name':
+      $human_name = t('N/A');
+      if (module_exists('workbench_moderation')) {
+        $state = $data->rel_entity_state;
+        $human_name = workbench_moderation_state_label($state);
+      }
+      break;
+
+    case 'rel_entity_bundle_name':
+      $type_info = entity_get_info($data->rel_entity_type);
+      $bundle = $data->rel_entity_bundle;
+      $bundle_info = $type_info['bundles'][$bundle];
+      $human_name = (isset($bundle_info['label'])) ? $bundle_info['label'] : $bundle;
+      break;
+
+    case 'rel_entity_type_name':
+      $type_info = entity_get_info($data->rel_entity_type);
+      $human_name = $type_info['label'];
+      break;
+  }
+
+  return $human_name;
+}
+
+/**
+ * Callback for getting netc_info related entity URL.
+ *
+ * @see nexteuropa_trackedchanges_entity_property_info_alter()
+ * @see hook_entity_property_info()
+ */
+function nexteuropa_trackedchanges_get_rel_uri($data, array $options, $name, $type, $info) {
+  $rel_entities = entity_load($data->rel_entity_type, array($data->rel_entity_id));
+  $uri = '';
+  if ($rel_entities) {
+    $rel_entity = reset($rel_entities);
+    $uri = entity_uri($data->rel_entity_type, $rel_entity);
+  }
+
+  return $uri ? url($uri['path'], $uri['options'] + $options) : '';
+}
+
+/**
+ * Get the 'nexteuropa_trackedchanges_info' entities related to an entity.
+ *
+ * @param string $entity_type
+ *   The type of the related entity.
+ * @param object $entity
+ *   The related entity.
+ *
+ * @return bool|array
+ *   An array of netc_info objects indexed by their ids.
+ *   When no results are found, FALSE is returned.
+ *
+ * @throws EntityMalformedException
+ *   The exception is thrown if $entity is malformed or if $entity_type
+ *   is empty.
+ */
+function nexteuropa_trackedchanges_info_load_by_entity($entity_type, $entity) {
+  if (!$entity_type) {
+    throw new EntityMalformedException('No entity type are set for this entity, 
+     the "nexteuropa_trackedchanges_info" cannot be retrieved.');
+  }
+
+  if (is_object($entity)) {
+    list($entity_id, , $bundle) = entity_extract_ids($entity_type, $entity);
+
+    if (!$entity_id) {
+      throw new EntityMalformedException('The entity has no id, the 
+        "nexteuropa_trackedchanges_info" cannot be retrieved.');
+    }
+
+    $query = new EntityFieldQuery();
+    $query->entityCondition('entity_type', 'netc_info');
+    $query->propertyCondition('rel_entity_id', $entity_id);
+    if ($bundle) {
+      $query->propertyCondition('rel_entity_bundle', $bundle);
+    }
+    $query->propertyCondition('rel_entity_type', $entity_type);
+    $results = $query->execute();
+
+    if (isset($results['netc_info'])) {
+      $info_ids = array_keys($results['netc_info']);
+      $info_entities = entity_load('netc_info', $info_ids);
```

Comment:
As we do not need `$info_ids`, it's not necessary to store it, use it directly in `entity_load()`.

`$info_entities = entity_load('netc_info', array_keys($results['netc_info']));`